server/src/plugins/tx-monitor.ts

```diff
@@ -0,0 +1,69 @@
+import fp from 'fastify-plugin';
+import { FastifyInstance } from 'fastify';
+import { IncomingMessage, ServerResponse, Server } from 'http';
+
+declare module 'fastify' {
+  export interface FastifyInstance<
+    HttpServer = Server,
+    HttpRequest = IncomingMessage,
+    HttpResponse = ServerResponse
+  > {
+    authenticate: any;
+  }
+}
+
+const cron = require('node-cron');
+const fetch = require('node-fetch');
+
+import { getPendingTxs, updateTransactionStatus } from '../db';
+import { TransactionStatus } from '../types';
+
+const EXPLORER_API_BASE = 'https://api.blockcypher.com/v1';
+
+export default fp(function transactionsMonitorCron(
+  fastify: FastifyInstance<Server, IncomingMessage, ServerResponse>,
+  opts,
+  next
+) {
+    const monitor = async () => {
```

Comment:
Hey @agusferreira we're not sticking to their Prettier config.

```json
{
  "printWidth": 100,
  "singleQuote": true,
  "trailingComma": "es5"
}
```

Even though `tabWidth` isn't defined here the default is 2 tabs, and you are using 4 tabs. 

You can run Prettier through their [CLI](https://prettier.io/docs/en/cli.html) or if you prefer to install the [VSCode](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) plugin and activate format on save on VSCode: `"editor.formatOnSave": true,`